sky/server/requests/requests.py

```diff
@@ -509,16 +520,29 @@ def get_request_tasks(
     cluster_names: Optional[List[str]] = None,
     exclude_request_names: Optional[List[str]] = None,
     user_id: Optional[str] = None,
+    include_request_names: Optional[List[str]] = None,
```

Comment:
minor: let's move the `exclude_xx` down, so the `include_` and `exclude_` are close to each other.